From b10d1a2701badc6f1cf118500aab9e46a1ec0a32 Mon Sep 17 00:00:00 2001
From: Matt Selsky <matthew.selsky@twosigma.com>
Date: Wed, 7 Dec 2016 10:09:14 -0500
Subject: [PATCH 205/268] Check for getpassphrase() no longer needed since it
 was used for C ntpq

Related to "Drop C version of ntpq in favor of the Python version."
---
 wafhelpers/configure.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/wafhelpers/configure.py b/wafhelpers/configure.py
index d86ba87..4968dd1 100644
--- a/wafhelpers/configure.py
+++ b/wafhelpers/configure.py
@@ -308,7 +308,6 @@ def cmd_configure(ctx, config):
                 ('clock_settime', ["time.h"], "RT"),
                 ('EVP_MD_do_all_sorted', ["openssl/evp.h"], "CRYPTO"),
                 ('getdtablesize', ["unistd.h"]),
-                ('getpassphrase', ["stdlib.h"]),                # Sun systems
                 ('getrusage', ["sys/time.h", "sys/resource.h"]),
                 ('MD5Init', ["md5.h"], "CRYPTO"),
                 ('ntp_adjtime', ["sys/time.h", "sys/timex.h"]),         # BSD
-- 
2.7.4

